test/Backends/Michelson.hs

```diff
@@ -449,18 +385,11 @@ xtwice =
           $ J.LamM [] ["y", "x", "z"]
           $ Ann one (primTy int)
           $ J.AppM
-            ( Ann
-                one
-                ( J.Pi one (primTy int)
-                    $ J.Pi one (primTy int)
-                    $ primTy int
-                )
+            ( Ann one (J.Pi one (primTy int) $ J.Pi one (primTy int) $ primTy int)
```

Comment:
I  prefer the old style for this, as it allowed the arguments to align with the $